scanner/src/main/kotlin/scanners/fossid/FossId.kt

```diff
@@ -151,6 +154,21 @@ class FossId(
             v.takeUnless { k in secretKeys }.orEmpty()
         }
 
+    private suspend fun getProject(projectCode: String): Project? =
+        service.getProject(user, apiKey, projectCode).run {
+            when {
+                error == null && data != null -> {
+                    FossId.log.info { "Project '$projectCode' exists." }
+                    data
+                }
+                status == 0 && error == "Project does not exist" -> {
```

Comment:
Just curious: Did you deliberately swap the order of checks here WRT to my proposal? Because I liked the fact to have the two `error` checks come first in both `when` cases.